db/migrations/0004_ticket_ticket_unique_movie_session_row_seat.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.2 on 2023-12-11 01:17
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_order'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Ticket',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('row', models.IntegerField()),
+                ('seat', models.IntegerField()),
+                ('movie_session', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.moviesession')),
+                ('order', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.order')),
+            ],
+        ),
+        migrations.AddConstraint(
+            model_name='ticket',
+            constraint=models.UniqueConstraint(fields=('movie_session', 'row', 'seat'), name='unique_movie_session_row_seat'),
+        ),
+    ]
```

Comment:
[This comment](https://github.com/mate-academy/py-tickets-and-orders/pull/559/files#r1423775043) is not resolved, there are 4 additional migrations. Make only one that will have all the necessary changes and include it in the project.